egs/wsj/s5/steps/nnet3/lstm/make_configs.py

```diff
@@ -48,7 +47,7 @@ def GetArgs():
                         help="For chain models, if nonzero, add a separate output for cross-entropy "
                         "regularization (with learning-rate-factor equal to the inverse of this)",
                         default=0.0)
-    parser.add_argument("--include-log-softmax", type=str, action=nnet3_train_lib.StrToBoolAction,
+    parser.add_argument("--include-log-softmax", type=str, action=common_train_lib.StrToBoolAction,
```

Comment:
including common_train_lib in the config generator is a bit confusing. You might want to move these common methods shared across all the python scripts to a common library. However as config generators will be deprecated in the next few weeks, you can choose to ignore this comment.